src/main/java/com/powsybl/openloadflow/sensi/DcSensitivityAnalysis.java

```diff
@@ -629,17 +630,27 @@ public void checkSensitivities(Network network, LfNetwork lfNetwork, List<Sensit
         }
     }
 
-    public void checkContingencies(LfNetwork lfNetwork, List<Contingency> contingencies) {
-        for (Contingency contingency : contingencies) {
-            for (ContingencyElement contingencyElement : contingency.getElements()) {
+    public void checkContingencies(LfNetwork lfNetwork, List<ContingencyContext> contingencies) {
+        for (ContingencyContext contingency : contingencies) {
+            for (ContingencyElement contingencyElement : contingency.getContingency().getElements()) {
```

Comment:
This check is not useful anymore as already done in `ContingencyContext` creation ?

Reply:
I don't know. ContingencyContext is now an util class, so someone else may want to modify it, and maybe stop throwing an error if the contingency is not a branch. But the SensitivityAnalysis should continue throwing an error

Reply:
Ok we can keep it